src/frontend/apps/markdown/MdxRenderer/constants.tsx

```diff
@@ -0,0 +1 @@
+export const debouncingTime = 1000; // wait in milliseconds before rendering
```

Comment:
1 second is too long. When we are using the interface we don't know what is happening. I tried with `500` and it's better. We can lower it to 350 ms IMO